src/components/Snackbar/SnackbarContainer.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC, useEffect, useState } from 'react';
+import { SnackbarContainerProps, SnackbarProps } from './Snackbar.types';
+import { Portal } from '../Portal';
+import { Snackbar } from './Snackbar';
+import { useSnack } from './snack';
+
+import styles from './snackbar.module.scss';
+
+export const SnackbarContainer: FC<SnackbarContainerProps> = ({
+    parent = document.body,
+}) => {
+    const [snacks, setSnacks] = useState<SnackbarProps[]>([]);
+    const { eat } = useSnack();
+
+    const addSnack = (e: CustomEvent<SnackbarProps>): void => {
+        setSnacks((s) => [...s, e.detail]);
+    };
+
+    const removeSnack = (e: CustomEvent<string>): void => {
+        setSnacks((s) => s.filter((snack) => snack.id !== e.detail));
+    };
+
+    useEffect(() => {
+        document.addEventListener('serveSnack', addSnack);
+        document.addEventListener('removeSnack', removeSnack);
+        return () => {
+            document.removeEventListener('serveSnack', addSnack);
+            document.removeEventListener('removeSnack', removeSnack);
+        };
+    }, []);
+
+    const getSnacks = (): JSX.Element => (
+        <div className={styles.snackbarContainer}>
```

Comment:
Should SnackBar be treated as a modal from an A11y perspective? Say a SnackBar appears, should it take focus and read the content to the person using the app via some focus handling and arialabeledby?

Reply:
Another approach would be to make the content focusable?